tools/rosbag/src/rosbag/bag.py

```diff
@@ -75,24 +75,19 @@ class ROSBagException(Exception):
     """
     Base class for exceptions in rosbag.
     """
-    def __init__(self, value):
-        self.value = value
```

Comment:
Removing `value` will effectively change the public API of this exception. Therefore I don't this PR can be applied as-is since it would break compatibility with existing code.

Maybe the constructor should be kept and invoke the parent constructor explicitly? If the parent constructor is being called with `value` the custom `__str__` implementation might still be obsolete.